From 97623b74c3c0ba109e353922f3227624429b425c Mon Sep 17 00:00:00 2001
From: avishalo <avi1.shalom@intel.com>
Date: Wed, 6 Apr 2016 18:00:51 -0700
Subject: [PATCH 381/639] Title: Remove Debug log message from connection
 tracking Issue: HSD: 246206 IssueType: Defect Symptom: Log is getting full
 RootCause: Message is printed continiously Impact: Log is being flooded

Signed-off-by: avishalo <avi1.shalom@intel.com>

Change-Id: If73c68a253fd08cf10009991d6c1de3131192458
---
 net/netfilter/core.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/net/netfilter/core.c b/net/netfilter/core.c
index 16f2992..31fe8ad 100644
--- a/net/netfilter/core.c
+++ b/net/netfilter/core.c
@@ -210,7 +210,6 @@ next_hook:
 			  (   MAC_ISMULTICAST (ptr_ethhdr, h_dest))  )
 		{
 			skip_pp_discard=1;
-			printk(KERN_DEBUG "NF_DROP :Multicast pkg, Do not create PP drop session\n" );
 		}
 	}
 
-- 
2.10.1

